apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/class-wp-rest-wpcom-block-editor-nux-status-controller.php

```diff
@@ -79,7 +79,7 @@ public function get_nux_status() {
 			$variant = 'tour';
 		}
 
-		if ( defined( 'IS_ATOMIC' ) && IS_ATOMIC ) {
+		if ( ! class_exists( '\WPForTeams' ) ) {
```

Comment:
This allows the check to work locally -- the only reason the atomic check exists is to prevent accessing a class that doesn't exist. (Same applies in a few other places)

Reply:
Edit: not adding this in this PR to derisk it. Maybe later.